R/emodnet_wfs.R

```diff
@@ -1,8 +1,67 @@
+emodnet_cache_file <- function() {
+  cache_dir <- rappdirs::user_cache_dir("EMODnetWFS")
+  if (!dir.exists(cache_dir)) {
+    dir.create(cache_dir, recursive = TRUE)
+  }
+
+  getOption("emodnet.services.csv") %||%
+    file.path(cache_dir, "services.csv")
+}
+
 .emodnet_wfs <- function() {
-  utils::read.csv(
-    system.file("services.csv", package = "EMODnetWFS"),
-    stringsAsFactors = FALSE
+  cache_file <- emodnet_cache_file()
+  # try reading from GitHub ----
+  url <- getOption("emodnet.services.url") %||%
+    "https://raw.githubusercontent.com/EMODnet/Web-Service-Documentation/salvadorf-use_readme_rmd/data/wfs.txt"
```

Comment:
@salvafern should we wait until your PR is merged until we merge?

Reply:
Probably yes..